robotoff/prediction/object_detection/core.py

```diff
@@ -156,18 +197,18 @@ def get_available_models(cls) -> List[str]:
 
     @classmethod
     def load_all(cls):
```

Comment:
I see this method is called at module importation (I find it unperfect already !).
Why do we do that, if it is meant to be served by triton ?

Reply:
We need the ID->label mapping (which is now in model directory in `labels.txt`), but it's indeed not necessary to load it at import time, we can load it lazily.